aspnetcore/fundamentals/host/hosted-services/samples-snapshot/2.x/BackgroundTasksSample-WebHost/Startup.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using BackgroundTasksSample.Services;
+using Microsoft.Extensions.Hosting;
+
+namespace BackgroundTasksSample
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+
+            #region snippet1
```

Comment:
Why aren't we using AddHostedService in here?

Reply:
It's a 2.1 thing, right? The "samples-snapshot" provides the doc with the 2.0 approach when the 2.0 version of the doc is loaded.

Reply:
Ah ok yes this is 2.1